containers/perl/devcontainer-template.json

```diff
@@ -0,0 +1,24 @@
+{
+	"id": "perl",
+	"version": "1.0.0",
+	"name": "Perl",
+	"description": "Develop Perl applications on Linux",
+	"documentationURL": "https://github.com/microsoft/vscode-dev-containers/tree/main/containers/perl",
+	"publisher": "Community",
+	"licenseURL": "https://github.com/microsoft/vscode-dev-containers/blob/main/LICENSE",
+	"options": {
```

Comment:
We can probably drop these given they're in the base image now. This predates the base I believe.